src/reducers/librarySlice.ts

```diff
@@ -66,6 +67,16 @@ export const initialState = (_chapter: t.Chapter | null): t.State => {
   };
 };
 
+export const fetchBooksThunk: AsyncThunk<void, null, RootState> = createAsyncThunk('library/fetchBooks', async (_payload, { dispatch, signal }) => {
+  const res = await fetch(`/books`, {
+    credentials: "include",
+    signal,
+  });
+
+  const { books } = await res.json();
+  dispatch(librarySlice.actions.setBooks(books));
```

Comment:
This is needed due to the fact that setBooks is a somewhat complex action, and therefore it did not feel appropriate to remove it and replace it in the below extraReducers section.

With that said, we can easily just put it below and remove the action. Doesn't make a difference.